frontend/src/pages/ImageDetails.tsx

```diff
@@ -125,6 +264,26 @@ function CreateImageDetailsPage() {
                   {(file.size != null) ? prettyBytes(file.size, { maximumFractionDigits: 2 }) : "0"}
                 </EuiText>
               </EuiFlexItem>
+              <EuiFlexItem >
+                <EuiText textAlign='center'>
+                  <strong>Hash:</strong>
+                </EuiText>
+                <EuiText textAlign='center'>
+                <EuiPopover
+                  isOpen={isPopoverOpen}
+                  button=
+                  {<EuiButtonEmpty
+                    
+                    color='text'                   
+                    onClick={() => {
+                      setIsPopoverOpen((isPopoverOpen) => !isPopoverOpen);
+                    }}>
+                      {file.sha256?.substring(0, 5)}...
+                  </EuiButtonEmpty>}>
+                      Hash: {file.sha256}
```

Comment:
When clicking the hash, if an image has more than one file then both of them open at the same time. Also, clicking elsewhere doesn't seem to close them and I'm not sure if that's desired or not.
![Capture](https://user-images.githubusercontent.com/70871525/227393108-dd24640d-bc2c-4ab7-aa42-5eff38c0ec50.PNG)